node-zerox/src/index.ts

```diff
@@ -51,9 +52,19 @@ export const zerox = async ({
   if (!localPath) throw "Failed to save file to local drive";
 
   const fileExtension = path.extname(localPath).toLowerCase();
+  let mimeType: string | null = null;
 
   if (!fileExtension) {
```

Comment:
What happens later in the code when `fileExtension` is missing? We are using to generate to `convertFileToPdf`

Reply:
If we try https://utfs.io/f/QlL5QaD0VUJEEiS9ZwCKPSVmOkY5TUrqIoaBzbdxJgcFRi3j
it will output 
```
Zerox error with https://utfs.io/f/QlL5QaD0VUJEEiS9ZwCKPSVmOkY5TUrqIoaBzbdxJgcFRi3j: Error: Unable to determine file type
```

Reply:
Let me try testing with that file.

Reply:
Seems to be coming through for me:
```
responsemimetypeeee application/rtf
extensionnnn rtf
mimetypeeee application/rtf
```

Reply:
Although looks like `binary/octet-stream` is not included in `mime-db`. So we might have to add a default for that case:
```
  let extension = mime.extension(mimetype);
  if (!extension) {
    if (mimetype === "binary/octet-stream") {
      extension = "bin";
    } else {
      throw new Error("File extension missing");
    }
    throw new Error("File extension missing");
  }
```